mesa/space.py

```diff
@@ -251,22 +251,33 @@ def get_neighborhood(
         if neighborhood is not None:
             return neighborhood
 
+        # Using a list for neighhborhood in respect to any other built-in data
+        # structure allows to optimize the code further (e.g. with Cython or Numba)
+        # more easily. Also, lists are more optimized in PyPy than in Cpython. Look at
+        # https://github.com/projectmesa/mesa/pull/1476#issuecomment-1306220403 and in
```

Comment:
Yeah, linking to this particular [comment](https://github.com/projectmesa/mesa/pull/1476#issuecomment-1306220403) is more informative.

Reply:
I think so too. Should something else be changed? :-)